OutSystems/outsystems-datagrid-reactive
# RGRIDT-684: Implementing API to retrieve invalid lines

This PR contains the fix for a bug where retrieving invalid lines was working after validating a cell on a row.


### What was done
* Fixed method call.

### Test Steps
1. Open sample
2. On ProductName column, delete the value of a cell
3. On Product category column choose laptop
4. Open console and execute the following command: JSON.parse(GridAPI.GridManager.GetChangesInGrid(GridAPI.GridManager.GetActiveGrid().uniqueId))
5. Turn the prodcut name cell valid.
6. Open console and execute the following command: JSON.parse(GridAPI.GridManager.GetChangesInGrid(GridAPI.GridManager.GetActiveGrid().uniqueId))

Expected: The above method should return an object with hasInvalidLines and invalidLinesJSON  properties. The hasInvalidLines property should be true and the invalidLinesJSON should contain the dataItem of the row that has invalid mark.

### Checklist
